java/ql/automodel/src/AutomodelFrameworkModeCharacteristics.qll

```diff
@@ -24,9 +24,17 @@ newtype JavaRelatedLocationType =
   ClassDoc()
 
 newtype TFrameworkModeEndpoint =
-  TExplicitParameter(Parameter p) or
+  TExplicitParameter(Parameter p) { not p.getType() instanceof PrimitiveType } or
```

Comment:
Should we include `BoxedType` and `NumberType` as well? Those three are the ones we usually exclude in most queries' default sanitizers (and don't model as sources/sinks except in very specific queries).

Reply:
Thank you for the suggestion! [2e5971b](https://github.com/github/codeql/pull/14849/commits/2e5971bb57f24d653bca845755a5b94df7709943)